src/components/Lotto.js

```diff
@@ -0,0 +1,49 @@
+import { Console } from "@woowacourse/mission-utils";
+import {
+  ERROR_MESSAGES,
+  LOTTO_MAX_NUMBER,
+  LOTTO_MIN_NUMBER,
+} from "../constant.js";
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#validate(numbers);
+    this.#numbers = numbers;
+  }
+
+  #validate(numbers) {
+    if (numbers.length !== 6) {
+      throw new Error(ERROR_MESSAGES.LENGTH);
+    }
+
+    const checkNumber = new Set(
+      numbers.filter((num) => typeof num === "number" && !isNaN(num))
+    );
+
+    if (checkNumber.size < numbers.length) {
+      throw new Error(ERROR_MESSAGES.DUPLICATED_NUMBER);
+    }
+
+    checkNumber.forEach((num) => {
+      if (
+        typeof num !== "number" ||
+        num < LOTTO_MIN_NUMBER ||
+        num > LOTTO_MAX_NUMBER
+      ) {
+        throw new Error(ERROR_MESSAGES.NOT_IN_RANGE);
+      }
+    });
+  }
+
+  get numbers() {
+    return this.#numbers;
+  }
```

Comment:
Airbnb JS Style Guide에서는 [getters/setters 지시자를 지양](https://github.com/parksb/javascript-style-guide?tab=readme-ov-file#accessors--no-getters-setters)하라고 안내하고 있어요.

현재 해당 로직에서는 테스트에서는 문제가 없겠지만 유지보수/사이드 이펙트가 있을 수 있으니 아래와 같이 변경해보시는건 어떨까요?
```suggestion
  getNumbers() {
    return this.#numbers;
  }
```